api/package.json

```
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
+    "@graphql-yoga/plugin-response-cache": "^3.3.0",
```

```suggestion
    "@graphql-yoga/plugin-response-cache": "3.3.0",
```

Maybe I should not pin,